doc/pmemkv/pmemkv.7.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: PMEMKV
+header: PMEMKV
+date: pmemkv version 0.8
+...
+
+[comment]: <> (Copyright 201(?)-2019, Intel Corporation)
```

Comment:
? ?

Reply:
yeah, this date - I don't have an idea which year it should be? The first commit of pmemkv? 2019?

Reply:
This is a copyright for the man page, which is a new thing in this repo, so 2019 is enough.